app/controllers/sessions_controller.rb

```diff
@@ -107,11 +107,9 @@ def create
           redirect_to profile_path,
                       alert: "That way to log in cannot be added because it is associated to an email address that is already in use!" # TODO i18n
         else
-          Rails.logger.fatal "IllegalState: OAuth data: #{request.env['omniauth.auth']}; " \
-                             "status: #{@handler_result.outputs[:status]}"
-          raise IllegalState, "SessionsCreate errors: #{@handler_result.errors.inspect
-                              }; Last exception: #{$!.inspect}; Exception backtrace: #{$@.inspect
-                              }; status: #{@handler_result.outputs[:status]}"
+          Rails.logger.fatal "IllegalState: OAuth data: #{request.env['omniauth.auth']
+                             }; status: #{@handler_result.outputs[:status]}"
+          redirect_to root_path, alert: 'Sorry, we lost you. Please log in again.'
```

Comment:
I think this should be more of the 500 error case where we show a 500 error and get an email, b/c we are not expecting users to get to this `else` block.